docs/contracts/functions.rst

```diff
@@ -438,7 +438,10 @@ operations as long as there is enough gas passed on to it.
             // results in test.x becoming == 1 and test.y becoming 1.
 
             // If someone sends Ether to that contract, the receive function in TestPayable will be called.
-            require(payable(test).send(2 ether));
+            // Since that function writes to storage, it takes more gas than is available with a
+            // simple ``send`` or ``transfer``. Because of that, we have to use a low-level call.
+            (success,) = address(test).call{value: 1}("");
```

Comment:
Isn't `value: 1` even less than what `send` and `transfer` forward?

Reply:
What do you mean?

Reply:
value: 1 means 1 wei is being sent, which is usually less than 1 gas, whereas send and transfer forward 2300 gas to the called receive/fallback function

Reply:
Ok. Sending 2 ether is a more practical example.